applications/solvers/multiphase/twoPhaseSystem/phaseModels/polydispersePhaseModel/polydispersePhaseModel.C

```diff
@@ -1023,9 +1023,11 @@ void Foam::polydispersePhaseModel::averageTransport
 
             fvScalarMatrix corrEqn
             (
-                ((*this)*rho() - quadrature_.moments()[1])
-               /fluid_.mesh().time().deltaT()
-              + meanM1Flux
+                (
+                    ((*this)*rho() - quadrature_.moments()[1])
+                   /fluid_.mesh().time().deltaT()
+                  + meanM1Flux
+                )/**pos((*this) - residualAlpha_)*/
```

Comment:
Looks like some commented code snuck in here :wink:

Reply:
Haha you are fast! Thanks Pete :-)

Reply:
@jheylmun could you comment on this change? It seems there is no difference between the old and the new code.

Reply:
The change is below this in the coefficient of the Laplacian term. I missed the deletion of this line.